dev-docs/bidders/pangle.md

```diff
@@ -78,3 +101,58 @@ pbjs.setConfig({
     }
 });
 ```
+
+### Config Demo
```

Comment:
I'm not sure this Prebid configuration code example belongs into this adapter documentation. It may get out dated and we have to maintain examples in different places.

Please keep all examples related to your adapter.